pkg/rancher-desktop/assets/scripts/nginx.conf

```diff
@@ -0,0 +1,175 @@
+worker_processes  auto;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    map_hash_bucket_size 128;
+    include       mime.types;
+    default_type  application/octet-stream;
+
+    log_format proxy escape=json
+      '{'
+        '"access_time":"$time_local",'
+        '"request":"$request",'
+        '"status":"$status",'
+        '"bytes_sent":"$body_bytes_sent",'
+        '"host":"$host",'
+        '"ssl_protocol":"$ssl_protocol",'
+        '"connect_host":"$connect_host",'
+        '"connect_port":"$connect_port",'
+      '}';
+
+    log_format mitm escape=json
+      '{'
+        '"access_time":"$time_local",'
+        '"method":"$request_method",'
+        '"uri":"$uri",'
+        '"status":"$status",'
+        '"bytes_sent":"$body_bytes_sent",'
+        '"upstream_response_time":"$upstream_response_time",'
+        '"host":"$host",'
+        '"http_host":"$http_host",'
+        '"upstream":"$upstream_addr"'
+      '}';
+
+    gzip off;
+
+    server {
+        listen 3128;
+        listen [::]:3128;
+        server_name proxy;
+
+        access_log logs/proxy.log proxy;
+
+        proxy_connect;
+        proxy_connect_allow all;
+        proxy_connect_address 127.0.0.1:3129;
+        proxy_max_temp_file_size 0;
+
+        # response non-CONNECT requests
+        location / {
+            add_header "Content-type" "text/plain" always;
+            return 200 "Yo docker, wazzup? Let's watch the game!\n";
```

Comment:
… interesting message.

Reply:
Good catch! 😄 You should never see the response, unless you curl the proxy directly, but I will change it to something more boring...

Reply:
Done